### DISCUSSION TITLE
Functionality of the sleep feature of JBC soldering iron

### DISCUSSION THREAD

**Discussion by Frosti385** [AxxAxx/AxxSolder#66] [General]
Hey, 

I have a question regarding the sleep feature of the soldering iron, when you rest the handle e.g. onto the station.
How does the feature itself work? Does the µC sense a connection to ground or to another voltage level when the connection is closed at the "handle-dock" or does it work in a totally different way? :D Are there any higher currents running in this case?
Reason I'm asking is, that I want to build a not-portable station where the portable version can snap on via Magnets. And an idea I had was to get the Connection to the handle-sense-port over one of the magnets and the screw on the case of the portable version.

Thanks in advance :)

**Comment by AxxAxx** [comment]
Hi! 
Yes, the MCU sense the S_sense input (which has an external pull-up R18). When the handle is in the stand this input is pulled low through the soldering iron tip. There is no big current and the signal should be OK to transfer over magnets. Nice idea!

**Reply by Frosti385** [reply]
Hey,
thanks for your Response. :) 
Perfect! I will try to build it this way :) Thanks :D
